lib/tasks/bulk_email.rake

```diff
@@ -11,4 +11,22 @@ namespace :bulk_email do
       DeliveryRequestWorker.perform_async_in_queue(id, queue: :delivery_immediate)
     end
   end
+
+  desc "Email all subscribers of the Brexit checker"
+  task brexit_subscribers: :environment do
+    brexit_lists = SubscriberList.where("subscriber_lists.tags->>'brexit_checklist_criteria' IS NOT NULL")
+    email_ids =
+      brexit_lists.flat_map do |list|
+        BulkSubscriberListEmailBuilder.call(
+          subject: ENV.fetch("SUBJECT"),
+          body: ENV.fetch("BODY"),
+          subscriber_lists: list,
```

Comment:
We should [pass all of the lists here](https://github.com/alphagov/email-alert-api/blob/master/app/builders/bulk_subscriber_list_email_builder.rb#L2), instead of iterating over them: if we don't, then e could end up sending multiple emails if someone has multiple checker subs. You could actually simplify the whole task by doing:

```
Rake::Task["bulk_email:for_lists"].invoke(*brexit_lists)
```

Reply:
Thanks Ben, that is way nicer.